tripy/tests/helper.py

```diff
@@ -564,16 +564,7 @@ def pretty_str_from_dict(dct):
                 locals_str += f"\n{obj}"
 
     def split_block_lines(title, contents, lang="python"):
-        line = block.splitlines()[1]
-        indentation = len(line) - len(line.lstrip())
-
-        out = (
-            indent(
-                format_contents(title, contents, lang),
-                prefix=" " * (indentation - 4),
-            )
-            + "\n\n"
-        )
+        out = indent(format_contents(title, contents, lang), prefix=" " * indentation) + "\n\n"
```

Comment:
This makes it so the indentation of the output/local var blocks match the indentation of the code block. I could leave a comment explaining, but it's quite obvious when you look at it in the code with full context.